src/field/FieldController.cpp

```diff
@@ -57,7 +55,7 @@ FieldController::FieldController(Domain const &domain):
         field_source_concentration(FieldType::RHO),
 
         // Fields for sight of boundaries
-        sight(FieldType::RHO, 1.0, 0, domain.get_size()) {
+        sight(FieldType::RHO, 1.0) {
 
     field_u.copyin();
```

Comment:
ll. 60-95 are unnecessary because the fields are already getting copied in their constructor, aren't they?